src/test/java/mate/academy/springbootintro/controller/CategoryControllerTest.java

```diff
@@ -127,14 +138,14 @@ void getAll_Categories_ReturnsAllCategories() throws Exception {
         // Given
         CategoryDto category1 = new CategoryDto(
                 CATEGORY_ID_1,
-                "Fiction",
-                "Fiction books"
+                CATEGORY_NAME_1,
+                CATEGORY_DESCRIPTION_1

```

Comment:
You should create and use something similar to `createBookDto` you created in BookControllerTest